browser/ui/views/brave_actions/brave_shields_action_view.cc

```diff
@@ -156,6 +156,11 @@ void BraveShieldsActionView::ButtonPressed() {
     return;
   }
 
+  auto* web_content = tab_strip_model_->GetActiveWebContents();
+  if (web_content && web_content->GetLastCommittedURL().SchemeIs("chrome")) {
```

Comment:
Let's use `content::kChromeUIScheme` instead of raw `"chrome"` string.
And, in addition to chrome scheme, I think we have more schemes to block like `file://`.
You can refer existing shields impls for finding all blocked scheme list?

Reply:
and this check could be done at the start of this method.

Reply:
there is a `bool [SchemeIsLocal]` - i'll look into it

Reply:
`SchemeIsLocal` is part of cr100: https://source.chromium.org/chromium/chromium/src/+/3b6e654f54715eab75556fc6643d2d2c517ddbc7

I've created a private function for now but can replace it when cr100 merges.